src/sentry/api/helpers/source_map_helper.py

```diff
@@ -37,8 +37,12 @@ def source_map_debug(project, event_id, exception_idx, frame_idx):
         # already mapped
         return SourceMapDebug()
 
+    if event.platform == "node" and frame.context_line:
```

Comment:
@roggenkemper do we need it for react native as well?

Reply:
i checked - the react native issue i was thinking of had react-native as the SDK but node as the platform so i think this solves it too